app/Actions/Network/MaterializeServerNetworkRules.php

```
@@ -0,0 +1,243 @@
+            ->values()
+            ->all();
+    }
+
+    /**
+     * @return array<int, array{ip: string, mask: int}>
+     */
+    private function sources(Network $network, Server $server): array
+    {
+        if ($network->type !== NetworkType::PROVIDER && $network->cidr !== null && $network->cidr !== '') {
+            return [[
+                'ip' => long2ip(Cidr::base($network->cidr)),
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Support\Cidr' in method 'sources'. (undefined)

(StaticAccess)